standards/certification/subjects-versions.md

```diff
@@ -0,0 +1,14 @@
+# Subjects and Versions
+
+The SCS Certification subjects are discrimitated in two dimensions. The first is the technology layer, which is either IaaS (infrastructure) or KaaS (Kubernetes). The second dimension is the level, which refer to the level of sovereignity, which is achieved by the certificate. There are three different levels:
```

Comment:
Type? discriminate --> discriminate

Is "discriminate" the right term here? Wouldn't be "split" or "differentiate" fit better?

Reply:
Likely! I will change it when it revise the text.